### PR TITLE
Add feature request #276

### LINKED ISSUE #276: Open recently accessed files
Another thing I miss from native apps is the ability to open the most recent file I was editing last session.

In macOS apps that's a submenu under File, called Open Recent... which contains a list of X recent documents.

Currently, when I open Pixelorama and choose File Open I have to hunt the file again.

### PR DESCRIPTION
**Pull request by dasimonde**
Closes #276.

### CODE REVIEW

**Comment by dasimonde** [comment]
It works, but i get sometimes the same error like in #331

**Comment by dasimonde** [comment]
I found out, that sometimes x_symmetry_axis.points has just the length 1
I get the error message in this block in the deserialize method:

`if dict.has("symmetry_points"):
	x_symmetry_point = dict.symmetry_points[0]
	y_symmetry_point = dict.symmetry_points[1]
	x_symmetry_axis.points[0].y = floor(y_symmetry_point / 2 + 1)
	x_symmetry_axis.points[1].y = floor(y_symmetry_point / 2 + 1)
	y_symmetry_axis.points[0].x = floor(x_symmetry_point / 2 + 1)
	y_symmetry_axis.points[1].x = floor(x_symmetry_point / 2 + 1)`

The adding of the second vector does not work in the _init method:

`if !x_symmetry_axis:
	x_symmetry_axis = SymmetryGuide.new()
	x_symmetry_axis.type = x_symmetry_axis.Types.HORIZONTAL
	x_symmetry_axis.project = self
	x_symmetry_axis.add_point(Vector2(-19999, y_symmetry_point))
	x_symmetry_axis.add_point(Vector2(19999, y_symmetry_point))
	Global.canvas.add_child(x_symmetry_axis)`

File: src/Main.gd [hunk 1]
```diff
@@ -136,13 +136,31 @@ func load_last_project() -> void:
 		var file_check := File.new()
 		if file_check.file_exists(file_path): # If yes then load the file
 			OpenSave.open_pxo_file(file_path)
+			
+			Global.save_project_to_recent_list(file_path)
```
Comment (OverloadedOrama) [inline src/Main.gd:140]: Everything looks good! Just a question though, is this line needed? We are already calling `Global.save_project_to_recent_list()` from `OpenSave.open_pxo_file()`, right? It seems like `save_project_to_recent_list()` is called twice here, or am I missing something?

**Review by OverloadedOrama — APPROVED** [review]

**Comment by OverloadedOrama** [comment]
> I found out, that sometimes x_symmetry_axis.points has just the length 1
> I get the error message in this block in the deserialize method:
> 
> ```gdscript
> if dict.has("symmetry_points"):
> 	x_symmetry_point = dict.symmetry_points[0]
> 	y_symmetry_point = dict.symmetry_points[1]
> 	x_symmetry_axis.points[0].y = floor(y_symmetry_point / 2 + 1)
> 	x_symmetry_axis.points[1].y = floor(y_symmetry_point / 2 + 1)
> 	y_symmetry_axis.points[0].x = floor(x_symmetry_point / 2 + 1)
> 	y_symmetry_axis.points[1].x = floor(x_symmetry_point / 2 + 1)
> ```
> 
> The adding of the second vector does not work in the _init method:
> 
> ```gdscript
> if !x_symmetry_axis:
> 	x_symmetry_axis = SymmetryGuide.new()
> 	x_symmetry_axis.type = x_symmetry_axis.Types.HORIZONTAL
> 	x_symmetry_axis.project = self
> 	x_symmetry_axis.add_point(Vector2(-19999, y_symmetry_point))
> 	x_symmetry_axis.add_point(Vector2(19999, y_symmetry_point))
> 	Global.canvas.add_child(x_symmetry_axis)
> ```

Yeah I have encountered this issue many times too. I'm not sure why this happens, which is why I made Issue #331. So I don't think the problems comes from this PR.

File: src/Main.gd (hunk 1 above)
Comment (dasimonde) [reply src/Main.gd]: Ups, I forgot that line to remove.

**Review by OverloadedOrama — APPROVED** [review]
Looks good, thank you!

**Pull request merged**